task-list/app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<!-- all partials start with a _ -->
+
+
+  <%= form_for @task do |f| %>
+  <%= f.label :title %>
+  <%= f.text_field :title %>
+  <%= f.label :description %>
+  <%= f.text_field :description %>
+  <%= collection_select(:task, :person_id, Person.all, :id, :name) %>
+  <%= f.submit %>
+  <% end %>
```

Comment:
woo hoooooo form_for! And even collection_select! I was super confused about how this worked for a long time. Good use of it :)